contrib/cf-remote/README.md

```diff
@@ -85,6 +85,14 @@ Note that this demo setup (`--demo`) is notoriously insecure.
 It has default passwords and open access controls.
 Don't use it in a production environment.
 
+### Specify an SSH key
+
+If you have more than one key in `~/.ssh` you may need to specify which key `cf-remote` is to use.
+
+```
+$ export CFREMOTE_SSH_KEY="~/.ssh/id_rsa.pub"
```

Comment:
`CF_REMOTE_SSH_KEY`*

Reply:
will pull, squash and push this change tomorrow.